utils/related_news.py

```
@@ -0,0 +1,46 @@
+    주어진 제목과 주제를 기반으로 관련 뉴스 기사를 TF-IDF 유사도로 검색합니다.
+    
+    Args:
+        title (str): 기준이 될 기사 제목
+        category (str): 검색할 기사 카테고리
+        
+    Returns:
+        list: 유사도 순으로 정렬된 상위 5개 관련 기사 리스트
+    """
+
+    # ✅ 예시 기사 리스트 (딕셔너리 리스트)
+    sql = "SELECT * FROM related_news where DATE = '" + get_correct_time() + "' AND category = '" + category + "';"
```

Pylint (3.3.7)

[convention] Line too long (115/100)

(C0301)